buildSrc/build.gradle.kts

```diff
@@ -5,23 +5,6 @@ plugins {
   id("com.diffplug.spotless") version "8.0.0"
 }
 
-if (!hasLauncherForJavaVersion(17)) {
-  throw GradleException(
-    "JDK 17 is required to build and gradle was unable to detect it on the system.  " +
-        "Please install it and see https://docs.gradle.org/current/userguide/toolchains.html#sec:auto_detection " +
-        "for details on how gradle detects java toolchains."
-  )
-}
```

Comment:
I don't think this is needed anymore now that we have the toolchain provisioning enabled it should automatically download and use the right Java version